pkg/policy/mapstate.go

```diff
@@ -858,28 +758,43 @@ func (ms *mapState) insertWithChanges(key Key, entry mapStateEntry, features pol
 // deleted.
 //
 // Incremental changes performed are recorded in 'changes'.
-func (ms *mapState) denyPreferredInsertWithChanges(newKey Key, newEntry mapStateEntry, features policyFeatures, changes ChangeState) {
```

Comment:
The godoc comment of [`(*mapState).revertChanges`](https://github.com/cilium/cilium/pull/36056/files#diff-171041eb8ba793ddbbd91b25b747b921c76878b41d78aabdc44a6633fea3409cR714) above and [`(*L4Filter).toMapState`](https://github.com/cilium/cilium/pull/36056/files#diff-d97453ecfcf9baa1157943e655c4e37306e9ac57daa1f7c7a67fcd37c2a82c4bL534) are still referring to `(*mapState).denyPreferredInsertWithChanges` and should probably be adjusted.

Reply:
Done, thanks!